tests/selenium_test.go

```diff
@@ -158,6 +158,9 @@ func TestPacketInjectionCapture(t *testing.T) {
 		return
 	}
 
+	// wait a bit to be sure that the capture is really started because of slow systems.
+	time.Sleep(5 * time.Second)
```

Comment:
Rather retry the Gremlin statement bellow

Reply:
I don't think so, as it is to inject and I don't want to inject twice or more

Reply:
I am fine with the above if it works even on very-slow systems ... if this is not the case I would still recommend polling (such as 10 retry / 1s delay) ... is fast on fast systems but will still wait a while to give slow systems a chance to come up.